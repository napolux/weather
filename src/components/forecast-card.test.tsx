import React from 'react';
import ForecastCard from './forecast-card';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <ForecastCard
                date="02/04/2020" 
                min={6.62} 
                max={13.06} 
                icon="01d" 
                humidity={37} 
                pressure={1011} 
                speed={1.54} 
                sunrise="07:00" 
                sunset="19:52"
            />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});