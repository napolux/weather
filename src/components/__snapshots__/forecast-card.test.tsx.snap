// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="ui card"
  onClick={[Function]}
>
  <div
    className="image"
  >
    <img
      src="http://openweathermap.org/img/wn/01d@2x.png"
    />
  </div>
  <div
    className="content"
  >
    <div
      className="header"
    >
      02/04/2020
    </div>
    <div
      className="description"
    >
      Temperatura minima: 
      6.62
       °C 
      <br />
      Temperatura massima: 
      13.06
       °C
    </div>
  </div>
</a>
`;
